### ISSUE TITLE
Unable to use Gryp from GitLab CI/CD

### ISSUE THREAD

**Issue by kuwv** [anchore/grype#1335]
**What happened**:

GitLab utilizes shells of container images to perform CI/CD jobs. The Grype container doesn't not provide any shells. While the distroless images do provide a focused base image for CLI images it prevents Grype from being used by all CI/CD platforms where it is intended.

**What you expected to happen**:

GitLab should be able to utilize a shell and perform any additional pre/post setup actions for Grype.

**Steps to reproduce the issue**:

Create a project with a pipeline on GitLab.com (or any CI/CD utilizing shells) and attempt to use Grype:

```
image:
  name: docker.io/anchore/syft:v0.82.0
  entrypoint: ''
```

**Anything else we need to know?**:
https://docs.gitlab.com/ee/ci/docker/using_docker_images.html
https://github.com/anchore/syft/issues/1858

**Comment by kzantow** [comment]
Hi @kuwv, we currently publish a `-debug` tag, in the format: `anchore/grype:<version>-debug` (introduced in [PR 1106](https://github.com/anchore/syft/pull/1106)). This should provide a shell that is compatible with GitLab. Would you be able to try this and see if it works for you?

**Comment by wagoodman** [comment]
We can do the same suggestion here https://github.com/anchore/syft/issues/1858#issuecomment-1692356552

**Comment by spiffcs** [comment]
Linked answer from https://github.com/anchore/syft/issues/1858#issuecomment-1776390542

I've also added a new grype job to that repo that shows how the debug image can be invoked

Let me know if you have other questions on this one and I can reopen if the images are still not functioning as intended

**Comment by kuwv** [comment]
Yeah, I'm installing from apk for now. It probably won't in siloed environments though.
